qiskit/ignis/verification/tomography/interface.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+
+"""
+Interface for the full state and process tomography flow
+"""
+
+import qiskit
+from qiskit import Aer
+from qiskit.quantum_info import state_fidelity
+from qiskit.tools.qi.qi import outer
+from qiskit.ignis.verification.tomography.fitters import fitter_data, \
+    state_cvx_fit, process_cvx_fit, state_mle_fit, process_mle_fit
+from .basis.circuits import state_tomography_circuits
+from .basis.circuits import process_tomography_circuits
+from .data import tomography_data
+
+
+def perform_state_tomography(circuit, measured_qubits,
+                             backend=None,
+                             fitter='cvx',
+                             ideal=True,
+                             fidelity=True,
+                             shots=5000,
+                             ):
+    """
+       Run the full state tomography flow:
+       1) Generate tomography circuits.
+       2) Run the circuits on the given backend
+       3) Extract from the results a fitter-friendly representation
+       4) Run fitter to obtain density operator
+       5) (optional) create an ideal result using a statevector simulator
+
+       Args:
+           circuit: the circuit generating the state to measure
+           measured_qubits: the qubits to be measured
+           backend: the backend on which the circuit is ran
+           fitter: the fitter used to obtain the density operator
+           ideal: whether to compute an ideal density operator
+           fidelity: whether to compute the fidelity between the tomography
+                     result and the ideal result
+           shots: how many shots to perform with the backend
+                  on each measurement
+
+       Returns:
+            if ideal and fidelity are False, returns the density operator
+            otherwise returns a dictionary with keys
+                'rho': the density operator
+                'ideal_psi': ideal statevector
+                'fidelity' the fidelity between rho and ideal_psi
+       """
+
+    if backend is None:
+        backend = Aer.get_backend('qasm_simulator')
+    tomography_circuits = state_tomography_circuits(circuit, measured_qubits)
+    job = qiskit.execute(tomography_circuits, backend, shots=shots)
+    data_results = tomography_data(job.result(), tomography_circuits)
```

Comment:
Chris changed these few lines to just `StateTomographyFitter` (see the notebook)

Reply:
Done.